app/templates/components/category-list.hbs

```diff
@@ -0,0 +1,7 @@
+{{yield}}
+<ul class="collection">
```

Comment:
What do you think about naming this class `categories` or something that would tell us more about the actual content?